14/umbraco-cms/tutorials/creating-a-basic-website/setting-the-navigation-menu.md

```diff
@@ -36,35 +38,35 @@ You now have the following snippet in your **Master** Template:
 </ul>
 ```
 
-This snippet will now need to be merged with the navigation above it.
+This snippet needs to be merged with the navigation above it.
 
-The `<ul>` tag needs to be wrapped inside the `<div class="container">` and `<nav>` tags, and the classes need to be added to the correct tags as well.
+9. Wrap the `<ul>` tag inside the `<div class="container">` and `<nav>` tags. Add the classes to the correct tags as well.
```

Comment:
"The correct classes" - is this something that is self explanatory, or perhaps something that needs some more details?
Like, will the new Umbraco user be able to figure out the correct classes? They might, I'm just wondering 💪 

```suggestion
9. Wrap the `<ul>` tag inside the `<div class="container">` and `<nav>` tags.
10. Add the classes to the correct tags as well.
```

Reply:
That's a good point. I will remove that line to avoid confusion. Since we are providing the code snippet it should be easy to see and follow.